internal/utils/validation.go

```diff
@@ -198,3 +198,23 @@ func ValidateURLScheme(link string, scheme ...string) error {
 
 	return nil
 }
+
+// ValidateNoHTMLNorJSNorCSS detects HTML, <script> tags, inline JavaScript, and CSS styles in a string
+func ValidateNoHTMLNorJSNorCSS(input string) error {
+	// Regular expressions to catch HTML tags, <script> tags, javascript: URIs, <style> tags, and inline style attributes
+	htmlPattern := regexp.MustCompile(`</?[a-z][\s\S]*>`)
```

Comment:
Should we add `(?i)` here as well to make it case-insensitive ?

Reply:
Thanks for catching that! Fixed in https://github.com/stellar/stellar-disbursement-platform-backend/pull/459/commits/56ee51dbe69bf2cea2ea6e125fb0daa19a990a98